### PR TITLE
JDP200201-11

### PR DESCRIPTION
**Pull request by zieluzieluzielu**
- Implementation of **Order** entity.
- Creation of the **Order** class. Adding relations.
- Modification/adaptation of **OrderDto** and **OrderController** classes
- I've added classes : **Product**, **User** to include the relations (those are not the final ones, to be filled by other colleagues)

### CODE REVIEW

File: src/main/java/com/kodilla/ecommercee/domain/Order.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    private Long orderId;
+
+    @Column(name = "USER_ID")
+    private Long userId;
+
+    @Column(name = "CREATION_DATE")
+    private Date created;
+
+    @Column(name = "SUM")
+    private Double sum;
```
Comment (JanHulanicki) [inline src/main/java/com/kodilla/ecommercee/domain/Order.java]: gdzies wychytalem ze pieniedzy nie nalezy trzymac w double

Comment (zieluzieluzielu) [reply src/main/java/com/kodilla/ecommercee/domain/Order.java]: modified to BigDecimal (as agreed on our weekly meeting)

File: src/main/java/com/kodilla/ecommercee/domain/Product.java [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+import java.util.ArrayList;
+import java.util.List;
+
+@AllArgsConstructor
+@NoArgsConstructor
+@Getter
+@Entity(name = "products")
+public class Product {
```
Comment (adrianpaz) [inline src/main/java/com/kodilla/ecommercee/domain/Product.java]: To jest task tylko encji order, wiec nie powinno to byc enxji Product. Wiem, ze w Order masz relacje, ale to omozesz jaj stworzyc i zakomentowac.

File: src/main/java/com/kodilla/ecommercee/domain/User.java [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+import java.util.ArrayList;
+import java.util.List;
+
+@AllArgsConstructor
+@NoArgsConstructor
+@Getter
+@Entity(name = "users")
+public class User {
```
Comment (adrianpaz) [inline src/main/java/com/kodilla/ecommercee/domain/User.java]: Tej encji tez tutaj nie powinno być.

**Comment by zieluzieluzielu** [comment]
- deleted entities: Product, User
- commented @ManyToOne part in Order entity

**Pull request merged**